### ISSUE TITLE
[A11Y] In the reference text area, there is an invalid state WAI-ARIA

### ISSUE THREAD

**Issue by Devika008** [pkp/pkp-lib#9357]
## Issue Description
// What is the nature of the issue? How does it impact users?

In the reference text area of the details page of new submission, an ARIA state or property has an invalid value. It may have been mistyped.

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question

1. Create a new submission
2. Go on the details page and enter references

## What application are you using?
 OJS, OMP or OPS version X.X.X
 
 OJS 3.4

## Stack used 

- Device: laptop
- OS: MacOS (Monterey)
- Browser: Chrome
- Version: 117

## Testing method
- Automated test: Siteimprove Accessibility Checker

## Action
// Describe action that can be taken to remediate the issue. 

The aria-invalid attribute accepts TRUE or FALSE as values. Review the field to confirm whether the attribute is really required and what should be the default value, e.g.: aria-invalid=false

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

<textarea id="citations-citationsRaw-control" name="citationsRaw" aria-describedby="citations-citationsRaw-description" aria-invalid="0" class="pkpFormField__input pkpFormField--textarea__input"></textarea>
